relay-metrics/src/aggregation.rs

```diff
@@ -1931,7 +1931,7 @@ impl AggregatorService {
         } = msg;
         for metric in metrics {
             if let Err(err) = self.insert(project_key, metric) {
-                relay_log::error!("failed to insert mertrics: {}", LogError(&err));
+                relay_log::error!("failed to insert metrics: {}", LogError(&err));
```

Comment:
nit: I think we should leave these changes out of scope for PRs that have a different goal and work on other parts.

Reply:
ok changed it back, but then should I create a separate PR just for tiny typos?